gamla/functional.py

```diff
@@ -259,10 +273,23 @@ def invoke(x):
 
 @currying.curry
 def assoc_in(d, keys, value, factory=dict):
+    """
+    Associate a new value to an existing dict given the path "keys".
```

Comment:
I think this function returns a new dict and not changing the given one (I'm not 100% sure)